attributes/default.rb

```diff
@@ -20,7 +20,7 @@
 default['kibana5']['url'] = nil
 default['kibana5']['checksum'] = nil
 
-default['kibana5']['version'] = '5.6.9'
+default['kibana5']['version'] = '6.3.0'
```

Comment:
this cookbook is called kibana5, so i expect to install 5th version by default. let's add support for 6, but let 5 stay default. what do you think?

Reply:
@anuriq About installing 5th version - totally agree, if we need, we can rename the cookbook and drop support of old kibana versions later on.

About old versions, if we continue to keep this cookbook upstream, they will stack due to backwards compatibility, so I think it will be more consistent to add whole stack of versions. Your opinion?